### PR TITLE
add some golang's builtin funcitons to go-mode

### PR DESCRIPTION
**Pull request by chunzhouyu**

### CODE REVIEW

**Comment by chunzhouyu** [comment]
hi @dgutov, please merge this commit, for adding some golang's builtin funcitons to go-mode

**Comment by stardiviner** [comment]
If gocode does not provide built-in functions and commands completion. I think `company-keywords` is for this purpose. Yes.

**Comment by chunzhouyu** [comment]
@stardiviner yeah, the gocode doesn't support golang builtin functions completion

File: company-keywords.el [hunk 1]
```diff
@@ -142,14 +142,15 @@
...
     (go-mode
-     ;; From https://golang.org/ref/spec#Keywords
-     ;; Plus some basic types, for better programming experience.
-     "bool" "break" "case" "chan" "complex128" "complex64" "const" "continue"
-     "default" "defer" "else" "fallthrough" "float32" "float64"
-     "for" "func" "go" "goto" "if" "import" "int" "int16" "int32"
-     "int64" "int8" "interface" "map" "package" "range"
-     "return" "select" "string" "struct" "switch" "type"
-     "uint" "uint16" "uint32" "uint64" "uint8" "uintptr" "var")
+     ;; 1. keywords ref: https://golang.org/ref/spec#Keywords
```
Comment (dgutov) [inline company-keywords.el]: Please capitalize the sentences.

**Comment by dgutov** [comment]
Thanks!

**Pull request merged**